docs/src/components/HeaderNavDropdown.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import NextLink from 'next/link';
+import { styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Collapse from '@material-ui/core/Collapse';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import IconButton from '@material-ui/core/IconButton';
+import SvgHambugerMenu from 'docs/src/icons/SvgHamburgerMenu';
+
+const Anchor = styled('a')(({ theme }) => ({
+  ...theme.typography.body2,
+  fontWeight: 700,
+  textDecoration: 'none',
+  color: theme.palette.text.secondary,
+  display: 'block',
+  padding: theme.spacing(1),
+  borderRadius: theme.spacing(1),
+  '&:hover, &:focus': {
+    backgroundColor: theme.palette.grey[200],
+  },
+}));
+
+export default function HeaderNavDropdown() {
+  const [open, setOpen] = React.useState(false);
+  const hambugerRef = React.useRef<HTMLButtonElement | null>(null);
+  return (
+    <React.Fragment>
+      <IconButton
+        ref={hambugerRef}
+        disableRipple
+        onClick={() => setOpen((value) => !value)}
+        sx={{
+          position: 'relative',
+          borderRadius: 1,
+          '&:focus': { boxShadow: `0 0 0 1px #e5e8ec` },
```

Comment:
Design token, I guess it will fail in dark mode

Reply:
I don't consider dark mode in this PR. I think it should be in another PR for dark mode.